doc/classes/EditorSettings.xml

```diff
@@ -477,6 +477,18 @@
 		<member name="filesystem/file_dialog/thumbnail_size" type="int" setter="" getter="">
 			The thumbnail size to use in the editor's file dialogs (in pixels). See also [member docks/filesystem/thumbnail_size].
 		</member>
+		<member name="filesystem/import/blender/blender3_path" type="String" setter="" getter="">
+			The path containing the Blender executable used for converting the Blender 3D scene files [code].blend[/code] to glTF 2.0 format during import. Blender 3.0 or later is required.
```

Comment:
```suggestion
			The path to the directory containing the Blender executable used for converting the Blender 3D scene files [code].blend[/code] to glTF 2.0 format during import. Blender 3.0 or later is required.
```
I believe this is more clear that it isn't the executable, which has caused confusion